crates/host/src/wasmbus/mod.rs

```diff
@@ -869,6 +869,12 @@ async fn create_lattice_metadata_bucket(
     jetstream: &JetstreamContext,
     bucket: &str,
 ) -> anyhow::Result<()> {
+    // Don't create the bucket if it already exists
```

Comment:
This technically introduces a race condition: consider the scenario of starting multiple hosts close to the same time/while another host/client is in the process of creating the bucket.

I think it's better to always try to create the bucket: if the settings are the same as what already exists, the NATS client will act like nothing bad happened. If the settings differ, the NATS client will return a relevant error

Reply:
The only issue with trying to create the bucket always is if the ability to create a stream is blocked at a NATS account level then the error we get back is that Jetstream is unavailable. There's no way that I could see to tell the difference from not being allowed to create a stream vs not having Jetstream connectivity.

I do agree with the small race condition though, and wouldn't mind a better way to handle this.

Reply:
Mm yeah good point. I think we'll have to accept this as a race condition then. The good news is even in most race conditions, multiple racing clients should be attempting to create the bucket with the same settings, so it shouldn't matter which wins. The only edge case is a scenario like:

1) client A (host) checks if the bucket exists -> false
2) client B creates the bucket with different settings (e.g. multiple replicas)
3) client A tries to create the bucket -> receives an error from the NATS client -> bails

In this case the recovery steps would be to restart the host, so it's not _that_ bad